snoop/data/analyzers/archives.py

```diff
@@ -59,6 +69,33 @@ def call_7z(archive_path, output_dir):
         raise ShaormaBroken("7z extraction failed", '7z_error')
 
 
+def unpack_mbox(mbox_path, output_dir):
+    def slice(stream):
+        last = b''
+        while True:
+            buffer = stream.read(1024 * 64)
+            if not buffer:
+                break
+            window = last + buffer
+            while True:
+                m = re.search(br'\n\r?\n(From )', window)
```

Comment:
Use case insensitive search here please.

Reply:
No, it's good.